math/errors.go

```diff
@@ -0,0 +1,21 @@
+package math
+
+import (
+	"errors"
+)
+
+// RootCodespace is the codespace for all errors defined in this package
+const RootCodespace = "math"
```

Comment:
Where is this used? The errors are normal errors

Reply:
this is unused, good catch